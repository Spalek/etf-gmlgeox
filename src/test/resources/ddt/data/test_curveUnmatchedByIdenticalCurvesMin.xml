<ii:GeometryCollection xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:ii="http://www.interactive-instruments.de/test" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 <gml:boundedBy>
  <gml:Envelope srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
   <gml:lowerCorner>0 0</gml:lowerCorner>
   <gml:upperCorner>90 90</gml:upperCorner>
  </gml:Envelope>
 </gml:boundedBy>
 <ii:member>
  <gml:CompositeCurve gml:id="c1">
   <gml:curveMember>
    <gml:Curve>
     <gml:segments>
      <gml:LineStringSegment>
       <gml:posList>2 1 2 2</gml:posList>
      </gml:LineStringSegment>
     </gml:segments>
    </gml:Curve>
   </gml:curveMember>
   <gml:curveMember>
    <gml:Curve>
     <gml:segments>
      <gml:LineStringSegment>
       <gml:posList>2 2 2 3</gml:posList>
      </gml:LineStringSegment>
     </gml:segments>
    </gml:Curve>
   </gml:curveMember>
  </gml:CompositeCurve>
 </ii:member>
 <ii:member>
  <gml:Curve gml:id="c2">
   <gml:segments>
    <gml:LineStringSegment>
     <gml:posList>2.5 5.5 1.5 5.5 1.5 4.5</gml:posList>
    </gml:LineStringSegment>
   </gml:segments>
  </gml:Curve>
 </ii:member>
 <ii:member>
  <gml:Curve gml:id="c3">
   <gml:segments>
    <gml:LineStringSegment>
     <gml:posList>4 6 4 5 4 4</gml:posList>
    </gml:LineStringSegment>
   </gml:segments>
  </gml:Curve>
 </ii:member>
 <ii:member>
  <gml:CompositeCurve gml:id="c4">
   <gml:curveMember>
    <gml:Curve>
     <gml:segments>
      <gml:LineStringSegment>
       <gml:posList>4.5 3 4.5 2</gml:posList>
      </gml:LineStringSegment>
     </gml:segments>
    </gml:Curve>
   </gml:curveMember>
   <gml:curveMember>
    <gml:Curve>
     <gml:segments>
      <gml:LineStringSegment>
       <!-- The following would be valid, since (4.5 1.5) is included: <gml:posList>4.5 2 4.5 1.5 4.5 0.5</gml:posList> -->
       <gml:posList>4.5 2 4.5 0.5</gml:posList>
      </gml:LineStringSegment>
     </gml:segments>
    </gml:Curve>
   </gml:curveMember>
  </gml:CompositeCurve>
 </ii:member>
 <ii:member>
  <gml:CompositeCurve gml:id="c5">
   <gml:curveMember>
    <gml:Curve>
     <gml:segments>
      <gml:ArcString>
       <gml:posList>6.5 2 7.5 3 8.5 2</gml:posList>
      </gml:ArcString>
     </gml:segments>
    </gml:Curve>
   </gml:curveMember>
   <gml:curveMember>
    <gml:Curve>
     <gml:segments>
      <gml:LineStringSegment>
       <gml:posList>8.5 2 8.5 1</gml:posList>
      </gml:LineStringSegment>
     </gml:segments>
    </gml:Curve>
   </gml:curveMember>
  </gml:CompositeCurve>
 </ii:member>
 <ii:member>
  <gml:Surface gml:id="s1">
   <gml:patches>
    <gml:PolygonPatch interpolation="planar">
     <gml:exterior>
      <gml:Ring>
       <gml:curveMember>
        <gml:Curve gml:id="s1_c1">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">1 1 2 1</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s1_c2">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">2 1 2 2</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s1_c3">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">2 2 1 2</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s1_c4">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">1 2 1 1</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
      </gml:Ring>
     </gml:exterior>
    </gml:PolygonPatch>
   </gml:patches>
  </gml:Surface>
 </ii:member>
 <ii:member>
  <gml:Surface gml:id="s2">
   <gml:patches>
    <gml:PolygonPatch interpolation="planar">
     <gml:exterior>
      <gml:Ring>
       <gml:curveMember>
        <gml:Curve gml:id="s2_c1">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">3 2 3 3</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s2_c2">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">3 3 2 3</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s2_c3">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">2 3 2 2</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s2_c4">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">2 2 3 2</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
      </gml:Ring>
     </gml:exterior>
    </gml:PolygonPatch>
   </gml:patches>
  </gml:Surface>
 </ii:member>
 <ii:member>
  <gml:Surface gml:id="s3">
   <gml:patches>
    <gml:PolygonPatch interpolation="planar">
     <gml:exterior>
      <gml:Ring>
       <gml:curveMember>
        <gml:Curve gml:id="s3_ec1">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">1 6 1 4</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s3_ec2">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">1 4 3 4</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s3_ec3">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">3 4 3 6</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s3_ec4">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">3 6 1 6</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
      </gml:Ring>
     </gml:exterior>
     <gml:interior>
      <gml:Ring>
       <gml:curveMember>
        <gml:Curve gml:id="s3_ic1">2.5 5.5 1.5 5.5 1.5 4.5 <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">2.5 5.5 2.5 4.5</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s3_ic2">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">2.5 4.5 1.5 4.5</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s3_ic3">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">1.5 4.5 1.5 5.5 2.5 5.5</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
      </gml:Ring>
     </gml:interior>
    </gml:PolygonPatch>
   </gml:patches>
  </gml:Surface>
 </ii:member>
 <ii:member>
  <gml:Surface gml:id="s4">
   <gml:patches>
    <gml:PolygonPatch interpolation="planar">
     <gml:exterior>
      <gml:Ring>
       <gml:curveMember>
        <gml:Curve gml:id="s4_c1">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">4 5 5 5</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s4_c2">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">5 5 5 6</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s4_c3">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">5 6 4 6</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s4_c4">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">4 6 4 5</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
      </gml:Ring>
     </gml:exterior>
    </gml:PolygonPatch>
   </gml:patches>
  </gml:Surface>
 </ii:member>
 <ii:member>
  <gml:Surface gml:id="s5">
   <gml:patches>
    <gml:PolygonPatch interpolation="planar">
     <gml:exterior>
      <gml:Ring>
       <gml:curveMember>
        <gml:Curve gml:id="s5_c1">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">4.5 0.5 5.5 0.5</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s5_c2">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">5.5 0.5 5.5 3</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s5_c3">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">5.5 3 4.5 3</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s5_c4">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">4.5 3 4.5 2</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s5_c5">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">4.5 2 4.5 1.5 4.5 0.5</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
      </gml:Ring>
     </gml:exterior>
    </gml:PolygonPatch>
   </gml:patches>
  </gml:Surface>
 </ii:member>
 <ii:member>
  <gml:Surface gml:id="s6">
   <gml:patches>
    <gml:PolygonPatch interpolation="planar">
     <gml:exterior>
      <gml:Ring>
       <gml:curveMember>
        <gml:Curve gml:id="s6_c1">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">6.5 2 6.5 1</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s6_c2">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">6.5 1 8.5 1</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s6_c3">
         <gml:segments>
          <gml:LineStringSegment interpolation="linear">
           <gml:posList count="2">8.5 1 8.5 2</gml:posList>
          </gml:LineStringSegment>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
       <gml:curveMember>
        <gml:Curve gml:id="s6_c4">
         <gml:segments>
          <gml:ArcString>
           <gml:posList count="2">8.5 2 7.5 3 6.5 2</gml:posList>
          </gml:ArcString>
         </gml:segments>
        </gml:Curve>
       </gml:curveMember>
      </gml:Ring>
     </gml:exterior>
    </gml:PolygonPatch>
   </gml:patches>
  </gml:Surface>
 </ii:member>
</ii:GeometryCollection>
