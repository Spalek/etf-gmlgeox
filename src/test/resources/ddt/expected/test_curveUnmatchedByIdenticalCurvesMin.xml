<test_curveUnmatchedByIdenticalCurvesMin>
  <test geom="c1">OK</test>
  <test geom="c2">OK</test>
  <test geom="c3">LINESTRING (4 6, 4 5, 4 4)</test>
  <test geom="c4">LINESTRING (4.5 2, 4.5 0.5)</test>
  <test geom="c5">OK</test>
</test_curveUnmatchedByIdenticalCurvesMin>